concept/type/AttributeType.java

```diff
@@ -41,6 +41,9 @@
     @Override
     FunctionalIterator<? extends Attribute> getInstances();
 
+    @Override
+    FunctionalIterator<? extends Attribute> getInstancesExplicit();
```

Comment:
we implement this new API following the style of our other "exact" methods like `getOwnsExplicit()`. This method retrieves instances of this type only, and excludes instances of subtypes.